runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -97,14 +113,36 @@ protected void startUp() throws Exception
 	protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
+		gameTime = null;
```

Comment:
set startedFromWaveOne to false here too. And hmm when at it also inGameBit to 0